src/lbaf/Utils/csv_2_json_data_converter.py

```diff
@@ -96,74 +93,89 @@ def _get_files_for_conversion(self) -> list:
+                    "obj_time": obj_time})
             else:
                 temp_dict[phase_id] = list()
-                temp_dict[phase_id].append({'phase_id': phase_id, 'obj_id': obj_id, 'obj_time': obj_time})
+                temp_dict[phase_id].append({
+                    "phase_id": phase_id,
+                    "obj_id": obj_id,
+                    "obj_time": obj_time})
 
+        # Return temporary dict
         return temp_dict
 
     def _write_json(self, output_path: str, data_to_convert: dict, node: int) -> None:
-        """ Method converts data to JSON and saves to output path
-        """
+        """ Convert data to JSON and saves to output path."""
         # Creating dictionary with right structure, which will be dumped
         dict_to_dump = dict()
-        dict_to_dump['phases'] = list()
-        for proc_id, others_list in data_to_convert.items():
-            phase_dict = {'tasks': list(), 'id': proc_id}
+        dict_to_dump["phases"] = list()
+        for rank_id, others_list in data_to_convert.items():
+            phase_dict = {"tasks": list(), "id": rank_id}
             for task in others_list:
-                task_dict = {'time': task['obj_time'], 'resource': 'cpu',
-                             'entity': {'id': task['obj_id'], 'type': 'object'}, 'node': node}
-                phase_dict['tasks'].append(task_dict)
-            dict_to_dump['phases'].append(phase_dict)
+                task_dict = {
+                    "time": task["obj_time"],
+                    "resource": "cpu",
+                    "entity": {
+                        "id": task["obj_id"],
+                        "type": "object"},
+                    "node": node}
+                phase_dict["tasks"].append(task_dict)
+            dict_to_dump["phases"].append(phase_dict)
 
         json_str = json.dumps(dict_to_dump, separators=(',', ':'))
         if self.compressed:
-            compressed_str = brotli.compress(string=json_str.encode('utf-8'), mode=brotli.MODE_TEXT)
-            with open(output_path, 'wb') as compr_json_file:
+            compressed_str = brotli.compress(string=json_str.encode("utf-8"), mode=brotli.MODE_TEXT)
+            with open(output_path, "wb") as compr_json_file:
                 compr_json_file.write(compressed_str)
         else:
-            with open(output_path, 'wt') as json_file:
+            with open(output_path, "wt") as json_file:
                 json_file.write(json_str)
 
     def main(self):
-        """ Main Class method. Gets lists of files to convert. Iterate over it and converts each file.
-        """
+        """ Get lists of files to convert. Iterate over it and converts each file."""
         files_to_convert = self._get_files_for_conversion()
         for file in files_to_convert:
             self._convert_file(file_path=file)
 
 
-if __name__ == '__main__':
-    Csv2JsonConverter(dir_path='data/vt_example_lb_data', compressed=False).main()
+if __name__ == "__main__":
```

Comment:
Does VT produces any CSV data anymore? I think `csv_2_json_data_converter.py` was created with purpose to used against `old` well known csv data in order to cenvert them to a new format. At the moment we use `hand made data` for testing purposes and it seems not to be relevant anymore.

Reply:
It doesn't produce CSV data anymore, but we do have some archived so this converter could potentially still be used.